vllm/v1/request.py

```diff
@@ -46,7 +46,7 @@ def __init__(
         self.num_prompt_tokens = len(self.prompt_token_ids)
         self._output_token_ids: List[int] = []
         self._all_token_ids: List[int] = self.prompt_token_ids.copy()
-        self.spec_token_ids: List[int] = []
+        self.spec_token_ids: Sequence[int] = []
```

Comment:
These changes are so the type checker is happy with empty tuples as well as lists.